internal/rbd/rbd_util.go

```diff
@@ -1089,7 +1097,7 @@ func (rv *rbdVolume) cloneRbdImageFromSnapshot(ctx context.Context, pSnapOpts *r
 	}
 
 	util.DebugLog(ctx, logMsg,
-		pSnapOpts, image, rv.imageFeatureSet.Names(), rv.Monitors)
+		pSnapOpts.String(), rv, rv.imageFeatureSet.Names(), rv.Monitors)
```

Comment:
there should not be a need to call `.String()`, calls to this function should be automatic with the formatting `%s` or `%q`. Other calls to `.String()` are removed where the object is a pointer to rbdVolume or rbdSnapshot.